packages/vue/src/components/molecules/SfNotification/SfNotification.html

```diff
@@ -0,0 +1,16 @@
+<transition :name="fade">
+    <div v-if="visible" @close="visible = false" class="sf-notification sf-notification__container" :class="`sf-notification--${type}`">
+        <slot name="icon" v-bind="{ icon, sizeIcon, colorIcon, type }">
+            <SfIcon class="sf-notification__icon" :icon="icon" :size="sizeIcon" :color="colorIcon" :class="`sf-notification--${type}`"></SfIcon>
+        </slot>
+        <header name="message" class="sf-notification__heading">
+            <component :is="`h${level}`" class="sf-notification__title">
```

Comment:
it should be just a title in div/span - a little bit overengineered ;)

Reply:
Should I use two differnt slots for title and the rest of the message or treat it as one?

Reply:
i think two

Reply:
but this comment is refering to dynamic component used here